infra/utils_test.py

```diff
@@ -55,7 +55,7 @@ def test_valid_filepath(self):
     """Tests that fuzz targets can be retrieved once the fuzzers are built."""
     fuzz_targets = utils.get_fuzz_targets(TEST_OUT_DIR)
     self.assertCountEqual(fuzz_targets,
-                          [os.path.join(TEST_OUT_DIR, 'do_stuff_fuzzer')])
+                          [os.path.join(TEST_OUT_DIR, 'example_crash_fuzzer'),os.path.join(TEST_OUT_DIR, 'example_nocrash_fuzzer')])
```

Comment:
this line is way too long, please fix. It's fine to use an additional local variable with a descriptive name in cases where code gets too hard to read otherwise

Reply:
Added.